pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -183,7 +187,7 @@ func NewForSchemas(client kube.Client, revision, domainSuffix string, schemas co
 			if _, f := known[name]; f {
 				handleCRDAdd(out, name, nil)
 			} else {
-				scope.Warnf("Skipping CRD %v as it is not present", s.Resource().GroupVersionKind())
+				scope.Warnf("%s:Skipping CRD %v as it is not present", identifier, s.Resource().GroupVersionKind())
```

Comment:
we can make `client.Scope = scope.WithLabel("controller", identifier)` and avoid needing to add this everywhere. Then just do `client.Scope.Warnf` etc